tb/main.cpp

```diff
@@ -13,11 +13,13 @@
  */
 
 #include <libtermbench/termbench.h>
-#include <iostream>
+
 #include <cstdio>
 #include <cstdlib>
+#include <format>
+#include <iostream>
 #include <string_view>
-#include <fmt/format.h>
+#include <fstream>
```

Comment:
shouldn't clang-format have alphabetically sorted this group of includes? 🤔

Reply:
fixed :)